src/server/action_handlers/actionkit-rsvp.js

```diff
@@ -0,0 +1,9 @@
+export const displayName = () => 'ActionKit Event RSVP'
+
+export const available = () => {
+  return true
+}
+
+export const processAction = (questionResponse) => {
```

Comment:
I'm going to suggest you pass in the campaign_contact object, and the campaign as well here (I assume it's already in the context sending out to it, so might as well send it in here for when it's useful.